srv/salt/_modules/cephdisks.py

```diff
@@ -155,7 +210,7 @@ def help_():
 
 __func_alias__ = {
     'all_': 'all',
-    'list_': 'list',
+    'used': 'list',
```

Comment:
Should `discover` also be an alias for `list` now?  Some might be using it.

Reply:
discover will be called from `disks.py` as a runner. I'll add a alias function there.